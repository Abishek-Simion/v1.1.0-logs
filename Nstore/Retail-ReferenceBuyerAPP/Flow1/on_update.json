{
    "context": {
        "ttl": "PT30S",
        "city": "std:080",
        "action": "on_update",
        "bap_id": "buyer-app-preprod.ondc.org",
        "bpp_id": "b-ondc-seller-bpp.nlincs.in",
        "domain": "nic2004:52110",
        "bap_uri": "https://buyer-app-preprod.ondc.org/protocol/v1",
        "bpp_uri": "https://b-ondc-seller-bpp.nlincs.in/bpp",
        "country": "IND",
        "timestamp": "2023-03-22T07:50:16.879Z",
        "message_id": "a930b94f-d62e-40b1-916d-73df89f0b58d",
        "core_version": "1.1.0",
        "transaction_id": "6f384034-4ed8-436c-ade0-81a3ca226099"
    },
    "message": {
        "order": {
            "id": "2023-03-22-460238",
            "items": [
                {
                    "id": "sm123",
                    "tags": {
                        "image": "",
                        "status": "Return_Initiated",
                        "reason_code": "003",
                        "update_type": "return",
                        "ttl_approval": "P2D",
                        "ttl_reverseqc": "P3D"
                    },
                    "quantity": {
                        "count": 1
                    },
                    "fulfillment_id": "20971"
                },
                {
                    "id": "sm124",
                    "price": {
                        "value": "75.00",
                        "currency": "INR"
                    },
                    "title": "Sakthi Biriyani Masala",
                    "quantity": {
                        "count": 1
                    },
                    "fulfillment_id": "20971",
                    "@ondc/org/title_type": "item"
                }
            ],
            "quote": {
                "ttl": "PT30M",
                "price": {
                    "value": "75.00",
                    "currency": "INR"
                },
                "breakup": [
                    {
                        "price": {
                            "value": "75.00",
                            "currency": "INR"
                        },
                        "title": "Sakthi Biriyani Masala",
                        "fulfillment_id": "20971",
                        "@ondc/org/item_id": "sm124",
                        "@ondc/org/title_type": "item",
                        "@ondc/org/item_quantity": {
                            "count": 1
                        }
                    }
                ]
            },
            "state": "Completed",
            "billing": {
                "name": "GirisutaA",
                "email": "girisuta@nstore.in",
                "phone": "8447407778",
                "address": {
                    "city": "Bengaluru",
                    "name": "GirisutaA",
                    "state": "Karnataka",
                    "country": "India",
                    "building": "Bethany school",
                    "locality": "8th Cross 5th Block Koramangala",
                    "area_code": "560005"
                },
                "created_at": "2023-03-22T07:47:14.000Z",
                "updated_at": "2023-03-22T07:48:25.000Z"
            },
            "payment": {
                "uri": "https://juspay.in/",
                "type": "ON-ORDER",
                "params": {
                    "amount": "75.00",
                    "currency": "INR",
                    "transaction_id": "ONDC-6f384034-4ed8-436c-ade0-81a3ca226099-1"
                },
                "status": "PAID",
                "tl_method": "http/get",
                "collected_by": "BAP",
                "@ondc/org/settlement_details": [
                    {
                        "bank_name": "IDFC FIRST BANK LIMITED",
                        "branch_name": "BKC Naman Chambers Branch",
                        "settlement_type": "neft",
                        "beneficiary_name": "NSTORE TECHNOLOGIES PRIVATE LIMITED",
                        "settlement_phase": "sale-amount",
                        "settlement_ifsc_code": "IDFCB0040101",
                        "settlement_counterparty": "seller-app",
                        "settlement_bank_account_no": "10112822953"
                    }
                ],
                "@ondc/org/buyer_app_finder_fee_type": "percent",
                "@ondc/org/buyer_app_finder_fee_amount": "3.0"
            },
            "provider": {
                "id": "20971",
                "locations": [
                    {
                        "id": "20971"
                    }
                ]
            },
            "documents": [
                {
                    "url": "https://receiptbeta.nstore.in/invoice/Mzg0ODc=",
                    "label": "Invoice"
                }
            ],
            "created_at": "2023-03-22T07:47:14.000Z",
            "updated_at": "2023-03-22T07:48:25.000Z",
            "fulfillments": [
                {
                    "id": "20971",
                    "end": {
                        "time": {
                            "range": {
                                "end": "2023-03-22T07:48:25.000Z",
                                "start": "2023-03-22T07:47:51.000Z"
                            },
                            "timestamp": "2023-03-22T07:48:25.000Z"
                        },
                        "contact": {
                            "phone": "8447407778"
                        },
                        "location": {
                            "gps": "13.0007580000001, 77.6165090000001",
                            "address": {
                                "city": "Bengaluru",
                                "name": "GirisutaA",
                                "state": "Karnataka",
                                "country": "India",
                                "building": "Bethany school",
                                "locality": "8th Cross 5th Block Koramangala",
                                "area_code": "560005"
                            }
                        },
                        "instructions": {}
                    },
                    "type": "Delivery",
                    "agent": {},
                    "start": {
                        "time": {
                            "range": {
                                "end": "2023-03-22T07:47:51.000Z",
                                "start": "2023-03-22T07:47:15.000Z"
                            },
                            "timestamp": "2023-03-22T07:47:51.000Z"
                        },
                        "contact": {
                            "email": "aleeminuk@gmail.com",
                            "phone": "9727288181"
                        },
                        "location": {
                            "id": "20971",
                            "gps": "12.9953231,77.6139826",
                            "descriptor": {
                                "name": "Evergreen SuperMarket",
                                "images": [
                                    "https://nstoreassets.s3.amazonaws.com/20971/store_logo/20971_b8j27qsnles4kuge.png"
                                ]
                            }
                        },
                        "instructions": {
                            "name": "",
                            "short_desc": ""
                        }
                    },
                    "state": {
                        "descriptor": {
                            "code": "Order-delivered"
                        }
                    },
                    "vehicle": {},
                    "tracking": false,
                    "@ondc/org/provider_name": "Evergreen SuperMarket"
                }
            ]
        }
    }
}