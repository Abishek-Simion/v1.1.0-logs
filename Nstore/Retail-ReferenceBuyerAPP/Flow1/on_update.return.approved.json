{
    "context": {
        "ttl": "PT30S",
        "city": "std:080",
        "action": "on_update",
        "bap_id": "buyer-app-preprod.ondc.org",
        "bpp_id": "b-ondc-seller-bpp.nlincs.in",
        "domain": "nic2004:52110",
        "bap_uri": "https://buyer-app-preprod.ondc.org/protocol/v1",
        "bpp_uri": "https://b-ondc-seller-bpp.nlincs.in/bpp",
        "country": "IND",
        "timestamp": "2023-03-15T12:31:07.818Z",
        "message_id": "4e656f98-04a3-4a24-94f7-c9c1d8086aed",
        "core_version": "1.1.0",
        "transaction_id": "aa2f2a49-4f44-41bb-8ba2-c78f68add300"
    },
    "message": {
        "order": {
            "id": "2023-03-15-887231",
            "items": [
                {
                    "id": "238",
                    "tags": {
                        "status": "Return_Approved",
                        "reason_code": 1,
                        "update_type": "return",
                        "ttl_approval": "P2D",
                        "ttl_reverseqc": "P3D"
                    },
                    "quantity": {
                        "count": 1
                    }
                },
                {
                    "id": "238",
                    "tags": {
                        "status": "Return_Approved",
                        "reason_code": 1,
                        "update_type": "return",
                        "ttl_approval": "P2D",
                        "ttl_reverseqc": "P3D"
                    },
                    "quantity": {
                        "count": 1
                    }
                },
                {
                    "id": "241",
                    "price": {
                        "value": "167.00",
                        "currency": "INR"
                    },
                    "title": "Saffola Classic Instant Masala Oats 500 g",
                    "quantity": {
                        "count": 1
                    },
                    "fulfillment_id": "20971",
                    "@ondc/org/title_type": "item"
                }
            ],
            "quote": {
                "ttl": "PT30M",
                "price": {
                    "value": "167.00",
                    "currency": "INR"
                },
                "breakup": [
                    {
                        "price": {
                            "value": "167.00",
                            "currency": "INR"
                        },
                        "title": "Saffola Classic Instant Masala Oats 500 g",
                        "fulfillment_id": "20971",
                        "@ondc/org/item_id": "241",
                        "@ondc/org/title_type": "item",
                        "@ondc/org/item_quantity": {
                            "count": 1
                        }
                    }
                ]
            },
            "state": "Completed",
            "billing": {
                "name": "GirisutaA",
                "email": "girisuta@nstore.in",
                "phone": "8447407778",
                "address": {
                    "city": "Bengaluru",
                    "name": "GirisutaA",
                    "state": "Karnataka",
                    "country": "India",
                    "building": "Bethany school",
                    "locality": "8th Cross 5th Block Koramangala",
                    "area_code": "560005"
                },
                "created_at": "2023-03-15T12:23:35.000Z",
                "updated_at": "2023-03-15T12:24:45.000Z"
            },
            "payment": {
                "uri": "",
                "type": "ON-ORDER",
                "params": {
                    "amount": "167.00",
                    "currency": "INR",
                    "transaction_id": ""
                },
                "status": "PAID",
                "tl_method": "http/get",
                "collected_by": "BAP",
                "@ondc/org/settlement_details": [
                    {
                        "bank_name": "IDFC FIRST BANK LIMITED",
                        "branch_name": "BKC Naman Chambers Branch",
                        "settlement_type": "neft",
                        "beneficiary_name": "NSTORE TECHNOLOGIES PRIVATE LIMITED",
                        "settlement_phase": "sale-amount",
                        "settlement_ifsc_code": "IDFCB0040101",
                        "settlement_counterparty": "seller-app",
                        "settlement_bank_account_no": "10112822953"
                    }
                ],
                "@ondc/org/buyer_app_finder_fee_type": "percent",
                "@ondc/org/buyer_app_finder_fee_amount": "3.0"
            },
            "provider": {
                "id": "20971",
                "locations": [
                    {
                        "id": "20971"
                    }
                ]
            },
            "documents": [
                {
                    "url": "https://receiptbeta.nstore.in/invoice/Mzg0MDY=",
                    "label": "Invoice"
                }
            ],
            "created_at": "2023-03-15T12:23:35.000Z",
            "updated_at": "2023-03-15T12:24:45.000Z",
            "fulfillments": [
                {
                    "id": "20971",
                    "end": {
                        "time": {
                            "range": {
                                "end": "2023-03-15T12:24:45.000Z",
                                "start": "2023-03-15T12:24:08.000Z"
                            },
                            "timestamp": "2023-03-15T12:24:45.000Z"
                        },
                        "contact": {
                            "phone": "8447407778"
                        },
                        "location": {
                            "gps": "13.0007580000001, 77.6165090000001",
                            "address": {
                                "city": "Bengaluru",
                                "name": "GirisutaA",
                                "state": "Karnataka",
                                "country": "India",
                                "building": "Bethany school",
                                "locality": "8th Cross 5th Block Koramangala",
                                "area_code": "560005"
                            }
                        },
                        "instructions": {}
                    },
                    "type": "Delivery",
                    "agent": {},
                    "start": {
                        "time": {
                            "range": {
                                "end": "2023-03-15T12:24:08.000Z",
                                "start": "2023-03-15T12:23:35.000Z"
                            },
                            "timestamp": "2023-03-15T12:24:08.000Z"
                        },
                        "contact": {
                            "email": "aleeminuk@gmail.com",
                            "phone": "9727288181"
                        },
                        "location": {
                            "id": "20971",
                            "gps": "12.9953231,77.6139826",
                            "descriptor": {
                                "name": "Evergreen SuperMarket",
                                "images": [
                                    "https://nstoreassets.s3.amazonaws.com/20971/store_logo/20971_b8j27qsnles4kuge.png"
                                ]
                            }
                        },
                        "instructions": {
                            "name": "",
                            "short_desc": ""
                        }
                    },
                    "state": {
                        "descriptor": {
                            "code": "Order-delivered",
                            "name": "Order-delivered"
                        }
                    },
                    "vehicle": {},
                    "tracking": false,
                    "@ondc/org/provider_name": "Evergreen SuperMarket"
                }
            ]
        }
    }
}